config/s2i/jenkins/jenkins-persistent-template.yml

```diff
@@ -2,7 +2,7 @@
 kind: Template
 apiVersion: v1
 metadata:
-  name: jenkins-persistent
+  name: "${JENKINS_SERVICE_NAME}-persistent"
```

Comment:
If this is a Jenkins template what would it be called besides Jenkins and where is the default name defined?

Reply:
On line 265, JENKINS_SERVICE_NAME is defaulted to jenkins. If it is hardcoded, then it is always jenkins and you can never have more than one jenkins masters in the same openshift project. I like to have jenkins-<project_name> and jenkins-<project_name>-stage. In some existing projects we have 3 separate masters already

Reply:
It's also good being able to differentiate between prod and other environments. I toggle the template label at the least for this reason.